layers/meta-tegrademo/conf/machine/jetson-nano-emmc-cot.conf

```diff
@@ -0,0 +1,15 @@
+MACHINEOVERRIDES = "jetson-nano-emmc:cot:${MACHINE}"
+require conf/machine/jetson-nano-emmc.conf
+PACKAGE_EXTRA_ARCHS_append = " jetson-nano-emmc"
+KERNEL_CLASSES += "tegra-kernel-fitimage"
+KERNEL_IMAGETYPE = "fitImage"
+TEGRA_ESSENTIAL_EXTRA_RDEPENDS_remove = "u-boot-extlinux"
+INITRAMFS_IMAGE_BUNDLE = "1"
+UBOOT_EXTLINUX = "0"
+UBOOT_ENTRYPOINT = "0x80080000"
+UBOOT_LOADADDRESS = "0x80080000"
+UBOOT_DTB_LOADADDRESS = "0x80000000"
+UBOOT_DTBO_LOADADDRESS = "0x80000000"
+UBOOT_RD_LOADADDRESS = "0x82a00000"
+UBOOT_RD_ENTRYPOINT = "0x82a00000"
+FIT_INCLUDE_DTBS = "0"
```

Comment:
Minor thing - missing a newline at the end of the file

Reply:
I will add it now

Reply:
Done